CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_quickbooks_source v0.12.0
+[PR #58](https://github.com/fivetran/dbt_quickbooks_source/pull/57) introduces the following updates:
+
+## Breaking Changes
+- We introduced these pre-existing timestamp fields from these sources to their equivalent staging models to better analyze real-time transaction data: 
```

Comment:
I would list the field and the exact staging models they were added to instead of requiring the user to have to match them up on their own. 

```suggestion
- Introduced the following timestamp fields to the listed staging models to better analyze real-time transaction data:
```

Reply:
@fivetran-catfritz Listing all the staging models would get a bit wordy, so I did a slight tweak to your suggestion. Let me know your thoughts.

Reply:
True--this makes sense!